bower.json

```diff
@@ -16,6 +16,7 @@
     "purescript-console": "^3.0.0",
     "purescript-math": "^2.0.0",
     "purescript-vector": "^2.0.0",
+    "purescript-matrix": "^2.0.0",  
```

Comment:
This library describes itself as a binding to the mjs library - does this mean that users of this library will need to install mjs via npm?